client/app.jsx

```diff
@@ -82,15 +117,66 @@ export default class App extends React.Component {
 
   handleLogIn(result) {
     const { user, token } = result;
+    fetch(`/api/account/get-maxes/${user.userId}`)
+      .then(res => res.json())
+      .then(result => {
+        const { benchMax, squatMax, deadliftMax, ohpMax, maxesUnit } = result;
+        this.setState({
+          user: user,
+          maxes: {
+            currentBench: benchMax,
+            currentSquat: squatMax,
+            currentDeadlift: deadliftMax,
+            currentOhp: ohpMax,
+            currentUnit: maxesUnit
+          }
+        });
+      });
     window.localStorage.setItem('react-context-jwt', token);
-    this.setState({ user });
   }
 
   handleFullModal() {
     const { isOpen } = this.state;
     this.setState({ isOpen: !isOpen });
   }
 
+  handleCreation(result) {
+    const { user } = this.state;
+    fetch(`/api/workout/get-workout/${user.userId}`)
+      .then(res => res.json())
+      .then(result => {
+        const tempWorkout = [];
+        let day = {};
+        let exercise = [];
+        let dayName = '';
+        for (let i = 0; i < result.length; i++) {
+          if (dayName !== result[i].workoutDayName) {
+            if (exercise.length > 0) {
+              day = {
+                workoutName: dayName,
+                exercise: exercise
+              };
+              tempWorkout.push(day);
+              day = {};
+              exercise = [];
+            }
+            dayName = result[i].workoutDayName;
+          }
+          if (dayName === result[i].workoutDayName) {
+            exercise.push(result[i]);
+          }
+          if (i === result.length - 1) {
+            day = {
+              workoutName: dayName,
+              exercise: exercise
+            };
+            tempWorkout.push(day);
+          }
+        }
```

Comment:
Same here. If the server processed the data you wouldn't have to do this in multiple parts of your client code.